README.md

```diff
@@ -2,21 +2,23 @@
 
-This repository creates a Docker image and publishes the image to DockerHub.
+This repository publishes a utility container for previewing and packaging [GOV.UK's Technical Documentation Template](https://github.com/alphagov/tech-docs-template).
 
-The Docker image is pulled by other repository CI/CD, uses the container with their source code and call the scripts with the container to create documentation websites that meet government technical documents. The data created, ie html website, is then pushed to GH Pages by the repository CI/CD.
+## ⚠️ Breaking Changes
 
-The contents of the Docker image will compile embedded ruby and markdown files, ie .html.md.erb, into HTML and assets using the GOV.UK [Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/) / [Source code](https://github.com/alphagov/tech-docs-template).
+### v4
 
-There are three scripts within the Docker image, that use the Middleman gem to either build or serve the data:
+The container is now hosted on GitHub Container Registry.
 
-- [preview.sh](scripts/preview.sh) - serve the compiled HTML and assets on a localhost port - useful for previewing the site locally
-- [deploy.sh](scripts/deploy.sh) - compiles the source code, places the HTML into the ./docs folder, tests the **internal links only** using htmlproofer and and creates a .tar artifact file that other GH Actions will serve to GitHub Pages.
-- [check-url-links.sh](scripts/check-url-links.sh) - compiles the source code, places the HTML into the ./docs folder, and tests **internal and external URL links** using htmlproofer.
+The testing framework is now [Lychee.](https://github.com/lycheeverse/lychee)
 
-This image is used by the [MoJ Template Documentation Site](https://github.com/ministryofjustice/template-documentation-site) repository for MOJ technical documentation that gets published to GitHub Pages.
+The scripts have moved:
 
-## Breaking Change in v3
+- `/scripts/deploy.sh` is now `src/usr/local/bin/td-package`
+
+- `/scripts/preview.sh` is now `src/usr/local/bin/td-preview`
```

Comment:
I'm not convinced this is an improvement on the current directory path. Why not just: `bin/preview`